src/control/server/engine/config.go

```diff
@@ -164,6 +164,25 @@ func (c *Config) Validate() error {
 		return errors.New("cannot specify both pinned_numa_node and first_core")
 	}
 
+	errNegative := func(s string) error {
+		return errors.New("memory size must not be negative")
+	}
```

Comment:
Looks like you're not using the string here.

Reply:
thanks, fixed